docs/components/SkipLinks.tsx

```diff
@@ -24,36 +23,37 @@ function SkipLink({ href, children, ...props }) {
         },
       }}
       {...props}
-    >
-      {children}
-    </a>
+    />
   );
 }
 
 export function SkipLinks() {
   const mq = useMediaQuery();
 
+  const skip = useCallback(
+    () => () => {
+      const skipTarget = document.getElementById('skip-link-navigation-btn');
+      skipTarget?.focus();
+    },
+    []
+  );
+
   return (
     <Fragment>
       <SkipLink
         href="#skip-link-navigation"
         css={{
-          display: ['none', null, 'block'],
+          display: ['none', undefined, 'block'],
         }}
       >
         Skip to Page Navigation
       </SkipLink>
       <SkipLink
         href="#skip-link-navigation-btn"
         css={mq({
-          display: ['block', null, 'none'],
+          display: ['block', undefined, 'none'],
```

Comment:
uhm ... why not `null`? That's the pattern we use everywhere where we want to keep the same value for that breakpoint?

Reply:
Technically (according to emotion's types) `null` isn't a valid value for this property.

Guessing it's one of those "browsers are forgiving" things, if you throw it a value it doesn't recognise it'll ignore it?

@mitchellhamilton might have an insight I don't on this

Reply:
Right. `null` isn't passed to the browser here. It just means the media query for this breakpoint is skipped. If `null` is not right we have to fix them all which should be a couple more places (I would estimate 2342348792834629346 places) ;)

Reply:
OK, good catch, mitchell pointed out the problem here is actually the missing `mq` calls which I missed. Fixed now.